resource/js/docready.js

```diff
@@ -844,7 +844,7 @@ $(function() { // DOCUMENT READY
       $('.search-result-listing').append($ready);
     }
     else {
-      $trigger.waypoint(function() { waypointCallback(); }, options);
+      $trigger.waypoint(function() { waypointCallback(this); }, options);
```

Comment:
The waypoint fixes here address issues when loading more search results on the search result page (possibly also on the alphabetical list in the sidebar) and the AJAX query for fetching more results fails for some reason.

Reply:
Tested this and the fix seems to be working.